core/lib/paths.ts

```diff
@@ -0,0 +1,10 @@
+type PubPath = `/c/${string}/pubs/${string}`;
+type PubEditPath = `/c/${string}/pubs/${string}/edit`;
+type PubCreatePath = `/c/${string}/pubs/create`;
+
+export const pubPath = (communitySlug: string, pubSlug: string): PubPath =>
+	`/c/${communitySlug}/pubs/${pubSlug}` as const;
+export const pubEditPath = (communitySlug: string, pubSlug: string): PubEditPath =>
+	`/c/${communitySlug}/pubs/${pubSlug}/edit` as const;
+export const pubCreatePath = (communitySlug: string): PubCreatePath =>
+	`/c/${communitySlug}/pubs/create` as const;
```

Comment:
ideally we'd use something more sophisticated for typed routes, but at least this makes linking to routes slightly less error prone